padinfo/core/transforminfo.py

```diff
@@ -8,20 +7,17 @@ async def perform_transforminfo_query(dgcog, raw_query, beta_id3):
     monster, err, debug_info = await findMonsterCustom2(dgcog, beta_id3, raw_query)
 
     if not monster:
```

Comment:
Can you rename `monster` to `found_monster`? Took me a second to understand why 21 & 23 were needed separately - I was going to ask you to leave a comment, but I think actually just this rename would be sufficient